tests/Gedmo/Sortable/Fixture/Node.php

```diff
@@ -15,7 +15,7 @@
 use Gedmo\Sortable\Entity\Repository\SortableRepository;
 
 /**
- * @author Charles J. C. Elling, 2017-07-31
```

Comment:
@mbabker, do you know what rule is changing these annotations?

BTW, I think we should separate this change in a different PR to confirm if it must be kept.